File: VB/DxBlazorApplication1/Services/Interfaces/ILabelService.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace DxBlazorApplication1.Services.Interfaces

    Public Interface ILabelService(Of TLabel)

        Public Function GetLabelsAsync() As Task(Of IList(Of TLabel))

    End Interface
End Namespace

File: CS/DxBlazorApplication1/Services/Interfaces/ILabelService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DxBlazorApplication1.Services.Interfaces {
    public interface ILabelService<TLabel> {
        public Task<IList<TLabel>> GetLabelsAsync();
    }
}
